### DISCUSSION TITLE
It's awesome that you guys are working on this.

### DISCUSSION THREAD

**Discussion by eitanavgil** [Fruktus/QuadradiusPreservationProject#27] [General]
I really loved this game and will be really happy to play it again. 
I was fantasizing of writing it, but it's way to complex for me to do it alone. If you plan to write the front-end of it from scratch, I will be happy to help (I am not a python guy, but I do know JavaScript, html, react etc')
Do you guys have a road map for this?
Cheers!

**Comment by Fruktus** [comment]
Hi! Thank you for your interest in game and kind words!

So a bit of explanation:
This repo is a server for the original flash version. In terms of gameplay it is complete, its missing few cosmetic features like scoreboard. It seems I should update the repo because almost everything here is outdated. The reason that this particular code is no longer updated is that it works, there is a little interest in it, I can't afford to host it and I don't own the copyrights.

The copyrights are actually the biggest problem right now, since even if I wanted to host everything and upload the game files (which are no longer available from original website) I would be liable.

This is a problem with the new version (my other repo called quadradiusR for remaster), where we already have mostly functional reimplementation but can't upload the beta versions or assets for the same reason.

This has kind of slowed us down and recently we didn't really do any significant progress.

To answer the roadmap question - as I said this repository is not actively updated for reasons above, but I can list all the things needed as issues and steps needed to fix them. The remaster does not have roadmap per se but instead has most of the tasks Todo written down and split by milestones (in the projects tab).

At this point I'm considering how to best approach the problem, because even if we recreated everything, we couldn't host it nor upload it via any official channels. Maybe we should just keep the original server for original version and redo the new one as full remake, with new features, graphics, in 3D etc.

If you have any suggestions or ideas let me know, any help and input is very much welcome :) 

Best regards
